crud/program.py

```
@@ -1,12 +1,33 @@
+from typing import List, Type
```

Ruff (0.8.2)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)